src/boot/index.js

```diff
@@ -4,15 +4,12 @@
 // The same boot script is used for both entry points so that the browser
 // already has it cached when it encounters the reference in the sidebar
 // application.
-
-import { parseJsonConfig } from './parse-json-config';
-
-import { bootHypothesisClient, bootSidebarApp } from './boot';
-import { processUrlTemplate } from './url-template';
-import { isBrowserSupported } from './browser-check';
-
 // @ts-ignore - This file is generated before the boot bundle is built.
```

Comment:
Can you add a blank comment line above this `// @ts-ignore`. The formatting plugin will delete any purely blank lines, but it will keep lines that start with just `//`.